wire-ios-utilities/Tests/PrivateUserDefaultsTests.swift

```diff
@@ -0,0 +1,88 @@
+//
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import XCTest
+@testable import WireUtilities
+
+final class PrivateUserDefaultsTests: XCTestCase {
+
+    var sut: PrivateUserDefaults<String>!
+    var userID: UUID!
+    let key = "foo"
+
+    // MARK: - Life cycle
+
+    override func setUp() {
+        super.setUp()
+        userID = UUID.create()
+        sut = PrivateUserDefaults(userID: userID, storage: .standard)
```

Comment:
use different storage for testing

Reply:
fixed in c8a59a522c5b2a06d5a9bf871a2a7e174175247d